docs/api/endorsement-provisioning/README.md

```diff
@@ -0,0 +1,104 @@
+# Endorsement Provisioning Interface
+
+The APIs described here are meant to illustrate the provisioning of 
+Endorsements based on RATS architecture using the Data Model as
+illustrated in 
+[Concise Reference Integrity Manifest](https://datatracker.ietf.org/doc/draft-birkholz-rats-corim/).
+
+## Provisioning API
+
+Provisioning API provides a facility to provision either Reference Values or Endorsed Values, using a 
+single CoRIM or a batch of CoRIM(s) in a single payload.
+
+* A client who intends to submit a CoRIM or a batch of CoRIM invokes a `POST` on a submit path;
+* If the transaction is complete then a `(200)`response is returned to the client. This is an indication
+  that the submitted CoRIM or batch has been successfully provisioned in a verifier. The ongoing transaction is now complete.
+* For large submission with multiple CoRIMs, the provisioning processing may happen in background. In this case,
+  Client is returned a `(202)` response, with a status of `processing`. Also 
+  client is returned with a session handle to subsequently poll the request for only a given amount of time.
+* The handle has a defined `time to live` upon its expiry automatic garbage collection is performed.
+* The session progresses as (`processing` -> `complete` | `failed`)
+* Client can continue periodic polling of the session uri, until the processing is either `complete` or `failed`
+
+
+## Synchronous submission
+
+```
+ o        (1) POST           .-------------.
+/|\ ------------------------>| /submit     |
+/ \ \ <----------------------|             |
+ \ \ \      200 (OK)         '-------------' 
+
+```
+
+## Asynchronous submission
+
+```
+ o        (1) POST           .-------------.
+/|\ ------------------------>| /submit     |
+/ \ \ <----------------------|             |
+ \ \ \      201 ( Session )  '-------------' 
```

Comment:
ok, yes, it was 201 :-)

Reply:
Correct, my bad, i have corrected in the text!